custom/style.css

```diff
@@ -90,12 +91,43 @@ code {
 }
 
 .box.warning {
-    border-left-color: #e7c000;
+    border-left-color: var(--c-warning);
     background-color: rgba(255, 229, 100, .2);
+    color: var(--c-warning-text);
+}
+
+.box.warning>.box-title{
+    color: var(--c-warning-title);
 }
 
 .spacer {
     width: 100%;
     height: 3px;
     margin: 25px 0px 25px 0px;
 }
+
+:root {
```

Comment:
Is it better to put those at the top, or the bottom of the file?

Reply:
I think bottom is best, I can add a comment describing the scope of that section if it's not clear enough.
E.g. "Global CSS variables", "Theme-relative styling overrides"

Reply:
Yeah, that'd be great!

Reply:
Okay, I've expanded a bit adding some comments. I've also moved the previous variable variable override in `:root` there, specifying the "override" (vs the other stuff that's added)